### ISSUE TITLE
Native player does not work with tv tuner plugin

### ISSUE THREAD

**Issue by mfisher11** [Rise-Vision/player-native#20]
Steps to reproduce:
1. Assign a presentation containing the Rise Display TV gadget (on a correctly configured player)

Result:
Presentation starts, but there is not content displayed from the TV gadget.
There is also a browser message at the top of the screen saying:
"this site uses a plugin (application/nprisetv2) that is unsupported."
Included is a link to a help page (screenshot below)

How it should work:
TV content should play in the gadget

Link to Presentation:
**THIS IS A CLIENT'S LIVE PRESENTATION. DO NOT MODIFY**
(providing this link because of the scarcity of tv gadget presentations)
http://rvauser.appspot.com/#/PRESENTATION_MANAGE/id=6ff27b23-e6ea-4b4b-91bd-6fbd28e2638d?cid=341bc8cf-e3f8-4f2c-a1b0-62c2ae6f977c

OS:
Windows 7

Chrome Ver:
Chrome 24.0.1312.56 (bundled)

![screenshot_2015-09-09_14-54-03](https://cloud.githubusercontent.com/assets/10407181/9773120/186bc5f4-5706-11e5-84c4-08e67bd9a78a.png)

**Comment by ByronDarlison** [comment]
@mfisher11 can you give me some context as to where this is coming from and why this broke all of a sudden and we need to support it? @alanclayton @btfrise ? Thanks

**Comment by mfisher11** [comment]
@ByronDarlison This was discovered when working on a ticket.  The existing installation (running the old native player) was crashing on startup so we upgraded to the new native player.  Unfortunately, the new version of player doesn't work with the plugin that handles communication between gadget and hardware.  According to the attached screenshot, it looks like Google decided to stop supporting that type of plugin last week.

**Comment by ByronDarlison** [comment]
@mfisher11 I am not completely sure I follow. What version of chrome is this appearing on? Version 24 is really old and it should work? @alanclayton can you review and let me know what the priority of this is? Also, @tejohnso @fjvallarino @btfrise I am worried that we may have found an issue that will break existing content that uses TV and if the auto deployment continues these will stop working. Can everyone confirm if the deployment should be halted until such time as these players are configured to stay on their current version of Chrome. Please respond asap. Thanks

**Comment by mfisher11** [comment]
@ByronDarlison The version was an error on my part (and I'll update the report once I finish this comment).  I had reinstalled the old version of player to get them back up and running before logging this bug--so when I copy/pasted the version reported in RVA, it was incorrect.  The version of chrome that has the problem is 44.0.2403.157.  @alanclayton @btfrise FYI

**Comment by alanclayton** [comment]
@ByronDarlison @fjvallarino @tejohnso we should halt the deployment while we look into this. @mfisher11 lets get together first thing tomorrow morning to go through this.

**Comment by tejohnso** [comment]
@ByronDarlison @alanclayton  The gradual deployment was halted last night.

 Is this the only thing holding us back to chrome 24?  It would be nice if we didn't have to support that version so that we could move our code to the modern age.  

We do have the option of honoring the "don't upgrade my browser" setting in the display set up, but that is currently not part of the new launcher / installer code.  I think that if a user wants to stay on an old browser it should be an unsupported option rather than something that we encourage.

**Comment by ByronDarlison** [comment]
@tejohnso agreed it is a pain in the ass and @alanclayton is looking into it, however, we just lost a piece of functionality that we currently have and we need to resolve that.  What's your points estimate for enabling how it worked previously? Thanks

**Comment by ByronDarlison** [comment]
@alanclayton do you have a card open to resolve this? To confirm, we have about 10,000 players that are currently being blocked from being upgraded by about 200 player with TV. Just to keep this in perspective. Thanks

**Comment by tejohnso** [comment]
@ByronDarlison enabling the browser upgrade prevention is a three, but it may be the case that fixing this problem for chrome 44 is also a three.

**Comment by ByronDarlison** [comment]
@tejohnso sorry, I don't follow, can you clarify this "but it may be the case that fixing this problem for chrome 44 is also a three". Which problem are you referring to? Thanks

**Comment by tejohnso** [comment]
@ByronDarlison the problem that the gadget doesn't work due to the nprisetv2 plugin using a deprecated feature.

**Comment by ByronDarlison** [comment]
@tejohnso ah ha. Thanks for the clarification. Let's leave that part to @alanclayton and the content team. Can you open a card for the first correction and let's address in the next sprint. Thanks!
